specification/configuration.md

```diff
@@ -130,6 +136,32 @@ beyond the OpenTelemetry specification exist.
+setting the following properties (in a language-specific way: Java may use builders, Swift can use named
+parameters with default values, JavaScript can use objects, etc.):
+
+| Configuration property (default value) | Description                                                                                                                                                                                                 | Required |
+| -------------------------------------- | -----------                                                                                                                                                                                                 | -------- |
+| `realm` ()                             | Splunk realm, e.g. `us0`, `us1`. If set, value of `beaconUrl` will be automatically computed based on this. [1] [2]                                                                                         | Yes      |
+| `beaconUrl` ()                         | RUM beacon URL, ex. `https://rum-ingest.<realm>.signalfx.com/v1/rum`. If both `realm` and `beaconUrl` are set, `beaconUrl` takes precedence. [1] [2]                                                        | Yes      |
+| `rumAuth` ()                           | RUM authentication token. [1]                                                                                                                                                                               | Yes      |
+| `applicationName` ()                   | Instrumented application name. [1]                                                                                                                                                                          | Yes      |
+| `globalAttributes` ()                  | OpenTelemetry [Attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/common.md#attributes) that will be added to every span produced by the RUM library. | Yes      |
+| `allowInsecureBeacon` (false)          | Allows sending data to insecure endpoints not using https. [2]                                                                                                                                              | No       |
+| `deploymentEnvironment` ()             | Sets the environment (`deployment.environment` span attribute) for all spans.                                                                                                                               | Yes      |
+
+- [1] Application name, authentication token and either realm or the beacon URL MUST be provided by the user.
+  If any of these is missing, the RUM instrumentation library MUST fail to start.
+- [2] Implementations MUST enforce by default that `beaconUrl`s are https only, and reject/fail to start otherwise.
+  Implementations MAY offer an `allowInsecureBeacon` option (default false) for customers wishing to use that.
+
+Other requirements:
```

Comment:
What about compression?

Reply:
Honestly I don't know. Are there any requirements around compression? @johnbley @jkwatson @jtmalinowski

Reply:
My understanding is that we do not compress today -- I am curious why and the cost perf/implications (which I assume are all server-side).

Reply:
There would be additional CPU usage on the client to do compression as well. For Android, we are just doing whatever the zipkin exporter does out of the box, which is whatever the zipkin client does out of the box.